src/plugins/data/public/search/search_interceptor.ts

```diff
@@ -98,7 +83,7 @@ export class SearchInterceptor {
   ): Observable<IEsSearchResponse> {
     const { id, ...searchRequest } = request;
     const path = trimEnd(`/internal/search/${strategy || ES_SEARCH_STRATEGY}/${id || ''}`, '/');
-    const body = JSON.stringify(id != null ? {} : searchRequest);
+    const body = JSON.stringify(id != null ? { stored: request.stored } : searchRequest);
```

Comment:
TODO: Need to make sure that `searchRequest` has the correct shape, rather than making these decidions in the OSS interceptor